src/NuGet.Clients/NuGet.Console/OutputConsole/OutputConsoleProvider.cs

```diff
@@ -65,12 +66,25 @@ public class OutputConsoleProvider : IOutputConsoleProvider
                         return new OutputConsole(vsOutputWindow, vsUIShell);
                     }
                 }, NuGetUIThreadHelper.JoinableTaskFactory);
+
+            _isServerMode = new AsyncLazy<bool>(
```

Comment:
Pretty sure it's safe here since there's no invocations of `_cachedOutputConsole` within the ctor, but, I'll ask.... should `_isServerMode` be initialized **before** it's used by `_cachedOutputConsole` async delegate, above?

Reply:
They are all lazy initializations so it doesn't matter. 
I can shift the order if you think it reads better, lmk.